### ISSUE TITLE
Cannot create VM in the demo machine

### ISSUE THREAD

**Issue by nertpinx** [kubevirt/demo#11]
After `make build` and `./run-demo.sh`, the next command fails:
```
[root@kubevirt-demo ~]# kubectl create -f /vm.json 
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```
Other things don't work as well, e.g. `virsh list --all`; checking libvirt it seems all configuration is lost (everything is left at default).

**Comment by fabiand** [comment]
The problem was probably that the api server was not up yet.

The reworked demo is checking this before returning.

**Comment by fabiand** [comment]
Fixed by #14
